---
Waypoint[] waypoints
uint8 status
